users/templates/users/register/password_reset_done.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'users/base.html' %}
+{% load crispy_forms_tags %}
+{% block main %}
+  <p>
+       We've emailed you instructions for setting your password 
+  </p>
+  <p>
+    If you don't receive an email, please make sure you've entered the address you registered with,
```

Comment:
If the email address isn't registered aren't you showing the error that it isn't registered? If yes then why is the user being confused here with this error message? Refactor this error message.

Reply:
changes done